src/machine/board_xiao.go

```diff
@@ -0,0 +1,135 @@
+// +build sam,atsamd21,xiao
+
+package machine
+
+import (
+	"device/sam"
+	"runtime/interrupt"
+)
+
+// used to reset into bootloader
+const RESET_MAGIC_VALUE = 0xf01669ef
+
+// GPIO Pins
+const (
+	D0  = PA02 // can be used for PWM or DAC
+	D1  = PA04 // PWM available
+	D2  = PA10 // PWM available
+	D3  = PA11 // PWM available
+	D4  = PA08 // can be used for PWM or I2C SDA
+	D5  = PA09 // can be used for PWM or I2C SCL
+	D6  = PB08 // can be used for PWM or UART1 TX
+	D7  = PB09 // can be used for PWM or UART1 RX
+	D8  = PA07 // can be used for PWM or SPI SCK
+	D9  = PA05 // can be used for PWM or SPI MISO
+	D10 = PA06 // can be used for PWM or SPI MOSI
+)
+
+// Analog pins
+const (
+	A0  = PA02 // ADC/AIN[0]
+	A1  = PA04 // ADC/AIN[4]
+	A2  = PA10 // ADC/AIN[18]
+	A3  = PA11 // ADC/AIN[19]
+	A4  = PA08 // ADC/AIN[16]
+	A5  = PA09 // ADC/AIN[17]
+	A6  = PB08 // ADC/AIN[2]
+	A7  = PB09 // ADC/AIN[3]
+	A8  = PA07 // ADC/AIN[7]
+	A9  = PA05 // ADC/AIN[6]
+	A10 = PA06 // ADC/AIN[5]
+)
+
+const (
+	LED         = PA17
+	PIN_LED_13  = PA17
+	PIN_LED     = PA17
+	LED_BUILTIN = PA17
+
+	PIN_LED_RXL = PA18
+	PIN_LED_TXL = PA19
+	PIN_LED2    = PIN_LED_RXL
+	PIN_LED3    = PIN_LED_TXL
```

Comment:
Can you match the TinyGo convention here?

```suggestion
	LED_RXL = PA18
	LED_TXL = PA19
	LED1    = LED
	LED2    = LED_RXL
	LED3    = LED_TXL
```

Reply:
Fixed at 14d94840e950d59847d15abc52a9f57db2dd1361